test/controllers/OtherReasonController.test.ts

```diff
@@ -51,11 +56,12 @@ describe('OtherReasonController', () => {
             const title = 'Some title';
             const description = 'Some description';
 
-            const app = createApplication(container => {
-                container.bind(RedisService).toConstantValue(createSubstituteOf<RedisService>(service => {
-                    service.set('session::other-reason', JSON.stringify({ title, description })).returns(Promise.resolve())
-                }))
-            });
+            const sessionStore = Substitute.for<SessionStore>();
```

Comment:
Please remove and put `{ title, description }` into session that is being passed to `createApp(session)` call.